docs/examples.md

```diff
@@ -41,6 +41,18 @@ check_android_task:
 !!! info
     Please don't forget to setup [Remote Build Cache](#build-cache) for your Gradle project. Or at least [simple folder caching](#gradle-caching).
 
+## Apache Maven
+
+Here is an example `.cirrus.yml` that builds a Maven project (and runs basic tests on code to make sure there are no simple errors):
```

Comment:
You can probably use similar working like this:

```suggestion
Official [Maven Docker images](https://hub.docker.com/_/maven/) can be used for building and testing Maven projects:
```

here to be more consistent.